src/components/UI/FormInputFile/FormInputFile.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { vitest } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { FieldError } from 'react-hook-form';
+import FormInputFile from './FormInputFile';
+import { InputType } from '../../../types/types';
+
+interface Props {
+  type: InputType,
+  name?: string,
+  title?: string,
+  error?: FieldError,
+  save: (file: string) => void,
+}
+describe('Component: FormSelect', () => {
+  it('should render error', () => {
```

Comment:
u can use just one describe for one component

Reply:
fixed